src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
Checking the prototype of MainWindow::open_project it returs a bool if opening fails.
The way we are exposing the method now, we loose the result of MainWindow::open_project and 
we don't have a way to know if opening was successful or if it failed.

Can we return a bool here?
Users can still retrieve the project from python using current_project().